assets/src/edit-story/app/story/effects/useLoadStory.js

```diff
@@ -48,8 +48,10 @@ function useLoadStory({ storyId, shouldLoad, restore }) {
           excerpt: { raw: excerpt },
           link,
           story_data: storyDataRaw,
+          // todo: get featured_media_url original dimensions for prepublish checklist
           featured_media: featuredMedia,
           featured_media_url: featuredMediaUrl,
+          // todo: get publisher_logo_url image dimensions for prepublish checklist
```

Comment:
Are these `todo` comments relevant still?

Reply:
yes, will be handled by #5105 & #5107